pkgs/shells/zsh/zsh-nix-shell/default.nix

```diff
@@ -14,16 +14,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Z6EYQdasvpl1P78poj9efnnLj7QQg13Me8x1Ryyw+dM=";
   };
 
-  strictDeps = true;
```

Comment:
Why is this removed?

Reply:
hmm it's not necessary but i'll add that so it causes no future errors